front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/pipeline/Pipeline.java

```diff
@@ -0,0 +1,118 @@
+package com.netflix.spinnaker.front50.model.pipeline;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Strings;
+import com.netflix.spinnaker.front50.model.Timestamped;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import org.codehaus.groovy.runtime.DefaultGroovyMethods;
+
+public class Pipeline extends HashMap<String, Object> implements Timestamped {
+
+  private static ObjectMapper MAPPER = new ObjectMapper();
+  public static final String TYPE_TEMPLATED = "templatedPipeline";
+
+  @JsonIgnore
+  public String getApplication() {
+    return (String) super.get("application");
+  }
+
+  @JsonIgnore
+  public String getName() {
+    return (String) super.get("name");
+  }
+
+  public void setName(String name) {
+    super.put("name", name);
+  }
+
+  @Override
+  @JsonIgnore
+  public String getId() {
+    return (String) super.get("id");
+  }
+
+  public void setId(String id) {
+    super.put("id", id);
+  }
+
+  @Override
+  @JsonIgnore
+  public Long getLastModified() {
+    String updateTs = DefaultGroovyMethods.asType(super.get("updateTs"), String.class);
```

Comment:
Should this be an untypedUtils thing (or something not DefaultGroovyMethods anyways)

Reply:
Oops. Missed this conversion. Thanks!